hardware/rtc.py

```diff
@@ -26,15 +26,17 @@
 
 def get_rtc():
     cmd = ['timedatectl', 'status', '--no-pager']
-    stdout = check_output(cmd).decode("utf-8")
+    try:
+        stdout = check_output(cmd).decode("utf-8")
+    except OSError:
+        LOG.warning('Unable to determine RTC timezone (no timedatectl)')
+        stdout = None
```

Comment:
Let's `return "unknown"`, otherwise you'll get a duplicate warning from line 41.

Reply:
Thanks, updated.